### ISSUE TITLE
Change `unsubscribe_configuration` method to accept List[str]

### ISSUE THREAD

**Issue by sarsharma** [dapr/python-sdk#488]
## Describe the proposal

The `unsubscribe_configuration` method currently accepts a single key to unsubscribe to 
https://github.com/dapr/python-sdk/blob/3d9b155af142fd531be95ceb0e2fccd237e630c2/dapr/clients/grpc/client.py#L976-L980 

Can it be changed to accept a list of strings to unsubscribe multiple config items in a single call

Also, the `get_configuration` method accepts a string of keys instead of `List[str]`, can the input `keys` be changed to `List[str]` to avoid confusion
https://github.com/dapr/python-sdk/blob/3d9b155af142fd531be95ceb0e2fccd237e630c2/dapr/clients/grpc/client.py#L905-L909

**Comment by sarsharma** [comment]
cc : @akhilac1

**Comment by berndverst** [comment]
Sure, but FYI, when you unsubscribe from one key it closes the stream and unsubscribes from all anyway. That's how the Dapr runtime works. @akhilac1 @sarsharma

**Comment by berndverst** [comment]
Please provide feedback as requested @akhilac1 @sarsharma.

It's not possible to unsubscribe from a single key - the stream will be closed. If anything I feel that we should not allow specifying any keys to unsubscribe from and just call it `unsubscribe_configuration(self, store_name)`

**Comment by berndverst** [comment]
Get configuration is fixed in #523 #491

**Comment by sarsharma** [comment]
> Please provide feedback as requested @akhilac1 @sarsharma.
> 
> It's not possible to unsubscribe from a single key - the stream will be closed. If anything I feel that we should not allow specifying any keys to unsubscribe from and just call it `unsubscribe_configuration(self, store_name)`

Makes sense! We could change the `unsubscribe_configuration()` method to be similar to Java, C#, Go SDKs which allow unsubscribe using the subscription id instead of key items.

**Comment by shivamkm07** [comment]
@berndverst can you please re-open this issue.
